mobile/mobile-core/android-process-service.ts

```diff
@@ -123,7 +169,7 @@ export class AndroidProcessService implements Mobile.IAndroidProcessService {
 			// USER     PID   PPID  VSIZE   RSS   WCHAN    PC         NAME
 			// u0_a63   25512 1334  1519560 96040 ffffffff f76a8f75 S com.telerik.appbuildertabstest
 			let processIdRegExp = /^\w*\s*(\d+)/;
-			let processIdInformation: string = adb.executeShellCommand(["ps", "|grep", "-a", "--text", appIdentifier]).wait();
+			let processIdInformation: string = adb.executeShellCommand(["ps", "|grep", appIdentifier]).wait();
```

Comment:
Commenting here as I cannot place a comment on the method's definition.
It seems this method is unused.